packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1666,9 +1664,11 @@ class StackViewport extends Viewport implements IStackViewport {
       // it in the space 3) restore the pan, zoom props.
       const cameraProps = this.getCamera();
 
-      this.panCache[0] = this.cameraPosOnRender[0] - cameraProps.position[0];
-      this.panCache[1] = this.cameraPosOnRender[1] - cameraProps.position[1];
-      this.panCache[2] = this.cameraPosOnRender[2] - cameraProps.position[2];
+      const panCache: Point3 = [
```

Comment:
This should be doable with a single line Mat3 operation - I think
const panCache = Mat3.subtract(Mat3.create(), this.cameraFocalPointOnRender, cameraProps.focalPoint)
It is too easy to get wrong when writing it all out by hand every time, would prefer to see it use the methods.